packages/core/package.json

```diff
@@ -6,15 +6,23 @@
   "scripts": {
     "build": "webpack"
   },
+  "files": [
+    "dist/*",
+    "*.js"
+  ],
   "main": "dist/main-bundle.js",
   "module": "src/main.js",
   "dependencies": {
     "@department-of-veterans-affairs/react-components": "workspace:*",
-    "@department-of-veterans-affairs/web-components": "workspace:*"
+    "@department-of-veterans-affairs/web-components": "workspace:*",
+    "react-focus-on": "^3.5.1",
+    "react-scroll": "^1.7.16",
+    "react-transition-group": "^1.0.0"
```

Comment:
These are used in `react-components`, but they don't actually get bundled. Adding them to `core` means that they do get bundled in the combined build - otherwise we get errors in a project like `vets-website`.

Reply:
Interesting. How was this working before?